Nightly build failing

[Nightly builds have started failing](https://github.com/aiidateam/aiida-quantumespresso/runs/1489788673?check_suite_focus=true) ever since the `asyncio` work was merged into `develop` of `aiida-core`. I don't understand the exception at all, but it essentially happens at the construction of a `Process`. Example:
```
=================================== FAILURES ===================================
___________________________ test_cp_autopilot[True] ____________________________

fixture_sandbox = <aiida.common.folders.SandboxFolder object at 0x7fd902e98710>
generate_calc_job = <function generate_calc_job.<locals>._generate_calc_job at 0x7fd902e8ed90>
generate_inputs_cp = <function generate_inputs_cp.<locals>._generate_inputs_cp at 0x7fd902e2e6a8>
file_regression = <pytest_regressions.file_regression.FileRegressionFixture object at 0x7fd902e98978>
autopilot = True

    @pytest.mark.parametrize('autopilot', [True, False])
    def test_cp_autopilot(fixture_sandbox, generate_calc_job, generate_inputs_cp, file_regression, autopilot):
        """Test a default `CpCalculation`."""
        entry_point_name = 'quantumespresso.cp'
    
        inputs = generate_inputs_cp(autopilot=autopilot)
>       calc_info = generate_calc_job(fixture_sandbox, entry_point_name, inputs)

tests/calculations/test_cp.py:14: 
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 
tests/conftest.py:121: in _generate_calc_job
    process = instantiate_process(runner, process_class, **inputs)
/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/aiida/engine/utils.py:59: in instantiate_process
    process = process_class(runner=runner, inputs=inputs)
/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/plumpy/base/state_machine.py:180: in __call__
    inst = super().__call__(*args, **kwargs)
/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/aiida/engine/processes/calcjobs/calcjob.py:124: in __init__
    super().__init__(*args, **kwargs)
/opt/hostedtoolcache/Python/3.6.12/x64/lib/python3.6/site-packages/aiida/engine/processes/process.py:115: in __init__
    communicator=self.runner.communicator)
_ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ _ 

self = <aiida_quantumespresso.calculations.cp.CpCalculation object at 0x7fd90d7ff438>
inputs = {'code': <Code: Remote code 'test.quantumespresso.cp' on localhost-test, pk: None, uuid: 0290bca5-6743-4c9a-99be-50ed2...27c-fde0db7f2388 (unstored)>, 'pseudos': {'Si': <UpfData: uuid: 7ad0a41e-cf0d-4cb7-90e7-943ef2222efb (unstored)>}, ...}
pid = None, logger = None
loop = <tornado.platform.asyncio.AsyncIOLoop object at 0x7fd902e47940>
communicator = <plumpy.communications.LoopCommunicator object at 0x7fd902cff358>

    def __init__(self, inputs=None, pid=None, logger=None, loop=None, communicator=None):
        """
        The signature of the constructor should not be changed by subclassing
        processes.
    
        :param inputs: A dictionary of the process inputs
        :type inputs: dict
        :param pid: The process ID, can be manually set, if not a unique pid
            will be chosen
        :param logger: An optional logger for the process to use
        :type logger: :class:`logging.Logger`
        :param loop: The event loop
        :param communicator: The (optional) communicator
        :type communicator: :class:`plumpy.Communicator`
        """
        super().__init__()
    
        # Don't allow the spec to be changed anymore
        self.spec().seal()
    
        self._loop = loop if loop is not None else asyncio.get_event_loop()
    
        self._setup_event_hooks()
    
        self._status = None  # May hold a current status message
        self._pre_paused_status = None  # Save status when a pause message replaces it, such that it can be restored
        self._paused = None
    
        # Input/output
        self._raw_inputs = None if inputs is None else utils.AttributesFrozendict(inputs)
        self._pid = pid
        self._parsed_inputs = None
        self._outputs = {}
        self._uuid = None
        self._creation_time = None
    
        # Runtime variables
>       self._future = persistence.SavableFuture(loop=self._loop)
E       AttributeError: 'AsyncIOLoop' object has no attribute 'get_debug'
```

--- a/.github/workflows/develop.yml
+++ b/.github/workflows/develop.yml
@@ -14,7 +14,7 @@ jobs:
 
         strategy:
             matrix:
-                python-version: [3.6, 3.7, 3.8]
+                python-version: [3.7, 3.8]
 
         services:
             postgres:
@@ -57,8 +57,8 @@ jobs:
             id: install
             run: |
                 pip install --upgrade setuptools
+                pip install git+https://github.com/aiidateam/aiida-core@develop#egg=aiida-core[atomic_tools]
                 pip install -e .[tests]
-                pip install git+https://github.com/aiidateam/aiida-core@develop
                 reentry scan
 
         -   name: Run pytest

